### PR TITLE
Add invalid urls

### PR DESCRIPTION
**Pull request by aljones15**
https://github.com/digitalbazaar/vc-data-model-2-test-suite/issues/5

Addresses the issue where URNs were being used as invalid URLs.

### CODE REVIEW

**Review by JSAssassin — APPROVED** [review]
Thanks, LGTM!

File: tests/input/credential-context-combo4-fail.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "@context": [
     "https://www.w3.org/ns/credentials/v2",
-    "urn:not:a:url"
+    "//not-a-url/contexts/example/v1"
```
Comment (davidlehn) [inline tests/input/credential-context-combo4-fail.json:4]: Why are all these not-URLs using a "//..." form?  Does that test some edge case code?  The simple string "not-a-url" would also fail.  That might be clearer and is more similar to the previous URN strings.  In practice we have seen attempts to use simple strings (like UUIDs) in place of URL ids, so it might be good to have a check like that in at least one test.

File: example-server.js [hunk 2]
```diff
@@ -95,12 +95,14 @@ function concatProof(existingProof, newProof) {
 }
 
 function validateUrl(value) {
-  if(value.startsWith('urn:')) {
-    return 'URN is not a URL';
-  }
   if(value.indexOf(':') === -1) {
```
Comment (davidlehn) [inline example-server.js:98]: Could use `includes` for this.  Is this check and error needed rather than the more generic `ERR_INVALID_URL` `TypeError` that would be caught below?
```suggestion
  if(!value.includes(':')) {
```

Comment (aljones15) [reply example-server.js:98]: This is more of a commentary on @clehner 's code and not part of this PR which turns the URNs to invalid urls. Maybe you should make an issue with code suggestions and code clean up for this project after the lint PR lands?

File: tests/input/credential-context-combo4-fail.json (hunk 1 above)
Comment (aljones15) [reply tests/input/credential-context-combo4-fail.json]: I actually kind of like a url failing because it has no protocol, but I went ahead and made the change here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/7/commits/b1b9517aed75add89e339a1d3d1ed3fb9b0d044c

Comment (davidlehn) [reply tests/input/credential-context-combo4-fail.json]: If it's a concern, tests can be added for variations of failure inputs.  I see you did "not-a-url/foo/bar".  I meant that the literal shorter "not-a-url" string would be enough to fail.  But it's fine either way.

**Review by davidlehn — APPROVED** [review]
